lms/static/sass/views/_account-settings.scss

```diff
@@ -36,45 +39,269 @@
+
+                a {
+                    text-decoration: none;
+                    color: $link-color;
+                }
+            }
+        }
+      }
     }
 }
 
 // +Settings Section
 .account-settings-sections {
+    padding-bottom: ($baseline*5);
 
     .section-header {
         @extend %t-title6;
         @extend %t-strong;
-        padding-bottom: ($baseline/2);
-        border-bottom: 1px solid $gray-l4;
+        padding-top: ($baseline/2)*3;
+        color: #5a5959;
     }
 
     .section {
         background-color: $white;
-        padding: $baseline;
         margin-top: $baseline;
-        border: 1px solid $gray-l4;
-        box-shadow: 0 0 1px 1px $shadow-l2;
-        border-radius: 5px;
-    }
+        border-bottom: 4px solid #f5f5f5;
 
-    a span {
-        color: $link-color;
-    }
+        .account-settings-header-subtitle {
+            font-size: em(18);
+            color: #7a7979;
+            padding-top: 20px;
+            padding-bottom: 10px;
+        }
+
+        .account-settings-section-body {
+
+            .u-field {
+                border-bottom: 2px solid #f5f5f5;
+
+                .field {
+                    width: 30%;
+                    vertical-align: top;
+                    display: inline-block;
+                    position: relative;
+
+                    select {
+                        -webkit-appearance: none;
+                        -moz-appearance: none;
+                        appearance: none;
+                        padding: 14px 30px 14px 15px;
+                        border: 1px solid #dae0e3;
+                        background-color: transparent;
+                        border-radius: 2px;
+                        position: relative;
+                        z-index: 10;
+                        &::-ms-expand{
+                            display: none;
+                        }
+                        ~ .icon-caret-down{
+                            &:after{
+                                content: "";
+                                border-left: 6px solid transparent;
+                                border-right: 6px solid transparent;
+                                border-top: 7px solid #0078be;
+                                position: absolute;
+                                right: 10px;
+                                bottom: 20px;
+                                z-index: 0;
+                            }
+                        }
+                    }
+                    .field-label {
+                        display: block;
+                        width: auto;
+                        margin-bottom: 0.625rem;
+                        font-size: 1rem;
+                        line-height: 100%;
+                        color: #7a7979;
+                    }
+
+                    .field-input {
+                        webkit-transition: all 0.125s ease-in-out 0s;
+                        transition: all 0.125s ease-in-out 0s;
```

Comment:
Please use the transition mixin already defined https://github.com/edx/edx-platform/blob/76b8e2e89761c30ac7aa2fa5e8aca4dc35d9002b/common/static/sass/bourbon/css3/_transition.scss#L6